ui/main/src/app/api/businessconfig.api.ts

```
@@ -0,0 +1,39 @@
+import {BusinessDataService} from 'app/business/services/businessconfig/businessdata.service';
+import {LogOption, LoggerService as logger} from '../business/services/logs/logger.service';
+
+declare const opfab: any;
+
+export class BusinessConfigAPI {
+    public static async getTags(screenName: string) {
+        return undefined;
+    }
+
+    public static init() {
+        const self = this;
```

Biome

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)